charts/timescaledb-single/templates/_helpers.tpl

```diff
@@ -103,13 +103,13 @@ ${HOME}/.pgbackrest_environment
 {{- end -}}
 
 {{- define "secrets_credentials" -}}
-{{ printf "%s-credentials" (include "clusterName" .) }}
+{{ printf "%s" (include "clusterName" .) }}
 {{- end -}}
 
 {{- define "secrets_certificate" -}}
-{{ printf "%s-certificate" (include "clusterName" .) }}
+{{ printf "%s" (include "clusterName" .) }}
 {{- end -}}
 
 {{- define "secrets_pgbackrest" -}}
-{{ printf "%s-pgbackrest" (include "clusterName" .) }}
+{{ printf "%s" (include "clusterName" .) }}
 {{- end -}}
```

Comment:
Wouldn't this try, by default, try to create 3 secrets with the same name and as such cause conflicts?

Reply:
Good point. I'm working with a fresh install scenario, and the changes required for that are pretty clear.

I've added another commit that (I think) uses the suffixes if there are no inputs. I'm not a Helm templating 🧙 so please sanity check that for me.